go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/Azure/go-autorest v11.7.1+incompatible
 	github.com/aws/aws-sdk-go v1.25.3
 	github.com/go-sql-driver/mysql v1.5.0
+	github.com/google/btree v1.0.0 // indirect
```

Comment:
Can we remove this?  I removed it locally and ran the tests and ran `make dev` and it was not re-added by go, so I think it's safe to remove

Reply:
removed.